test/integration/maincluster_test.go

```diff
@@ -1568,14 +1572,22 @@ worker:
 experimental:
   kiamSupport:
     enabled: true
+    image:
+      repo: quay.io/uswitch/kiam
+      tag: v2.6
```

Comment:
FYI @c-knowles, I believe that it is better to organize the image repo/tag settings like this, so that it is easier to migrate features like this into kube-aws plugins.